test/distributed/org/apache/cassandra/distributed/impl/AbstractCluster.java

```diff
@@ -1319,4 +1326,3 @@ private enum Utils implements NameHelper
         INSTANCE;
     }
 }
-
```

Comment:
@jacek-lewandowski same

Reply:
actually - no, I'll not revert that - I've just looked and in those two files there were 2 empty lines in the end. Now there is one

Reply:
ok thanks for the clarification